bogota-abril-2019/css-magazine-cover/david-villamizar/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8" />
+    <title>WIRED UK Amazon's Next Move</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link rel="manifest" href="site.webmanifest" />
+    <link rel="apple-touch-icon" href="icon.png" />
+    <!-- Place favicon.ico in the root directory -->
+
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/main.css" />
+
+    <meta name="theme-color" content="#fafafa" />
+  </head>
+
+  <body>
+    <!--[if IE]>
+      <p class="browserupgrade">
+        You are using an <strong>outdated</strong> browser. Please
+        <a href="https://browsehappy.com/">upgrade your browser</a> to improve
+        your experience and security.
+      </p>
+    <![endif]-->
+
+    <!-- Add your site or application content here -->
+    <div id="magazine-cover">
+      <img id="fake-jeff" src="img/fake-bezos.jpg" alt="Fake Jeff Bezos" />
+      <h1 id="magazine-title">
+        <img src="img/wired-logo.svg" alt="WIRED Logo" />
+        <span>
+          UK Edition
+        </span>
+      </h1>
+      <h2 id="issue-title">Amazon's Next Move</h2>
+      <section>
+        <h3>Exclusive</h3>
+        <h4>From Alexa to Drones:</h4>
+        <h5>The Jeff Bezos Master Plan</h5>
+      </section>
+      <section>
+        <span>Plus</span>
+        <hr />
+        <h3>Product Special</h3>
+        <h4>WIRED's Ultimate Gear Guide</h4>
+      </section>
+      <section>
+        <h4>Lab-made Diamonds</h4>
+        <h4>36 Essential Productivity Hacks</h4>
+        <h4>Cronuts 2.0!</h4>
```

Comment:
This h4 seems to be related.
Why there are 3 of them?

Reply:
Each one is the title of an article inside the magazine.

Reply:
I think it shouldn't have more than one h4. I wondering if you are trying to have here a nav menu.

Reply:
Notice the red lines on the lower left of the cover. It shows titles of articles, inside the magazine, should they be listed as `<li>`'s inside a nav maybe? If that's the case, the same could be said about other article titles in the cover.
![image](https://user-images.githubusercontent.com/4100566/56928675-326e2600-6a9d-11e9-9607-c5731356b042.png)